pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -231,16 +238,24 @@ def main(input, output, log_path, config_file, resources, processes):
     df[GENOMIC_HGVS_HG38_COL] = df[TMP_HGVS_HG38].apply(str)
 
     logging.info("Compute hg37 representation of internal representation")
-    var_objs_hg37 = convert_to_hg37(df[VAR_OBJ_FIELD], resources)
+    var_objs_hg37, var_objs_hg37_failed = convert_to_hg37(df[VAR_OBJ_FIELD], resources)
 
     logging.info("Compute hg37 normalized representation of internal")
     # normalizing again for the hg37 representation. An alternative would be to convert the normalized hg38 representation to hg37.
     # If we use crossmap, we would need a way to convert the VCF like representation back to an hgvs object, which we currently
     # are unable to do properly. That is, we can use VCFVariant.to_hgvs_obj, however, structural variants will be converted
     # to delins, losing information if a variant was e.g. a del, ins, or dup.
 
-    df[TMP_HGVS_HG37] = pd.Series(
-        [v.to_hgvs_obj(hgvs_proc.contig_maps[HgvsWrapper.GRCh37_Assem]) for v in var_objs_hg37])
+    # set None values for any hg37 coordinates that cannot be derived and add to lists in proper order
+    tmp_hgvs_hg37_values = [v.to_hgvs_obj(hgvs_proc.contig_maps[HgvsWrapper.GRCh37_Assem]) for v in var_objs_hg37]
```

Comment:
would be good to move this block _before_ the `logging.info("Compute hg37 normalized representation of internal")` line, as it's not about normalization but merely about computing a hg37 representation.
It would be also good to put this into a separate function for better readability of the main function.